sharedresource/install.go

```diff
@@ -0,0 +1,26 @@
+package sharedresource
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	sharedresourcev1alpha1 "github.com/openshift/api/sharedresource/v1alpha1"
+)
+
+const (
+	GroupName = "sharedresource.openshift.io"
```

Comment:
My only concern is currently the group name, seems very specific, don't you think? Why not creating a `storage.openshift.io` group or something like that instead?

Reply:
If you check our pre-existing groups (including k8s) you'll notice they usually express a broader categories, like apps, batch, or in openshift builds, images, etc.

Reply:
updated in next push